spec/ADAS/ADAS.vspec

```diff
@@ -200,6 +200,45 @@ ESC.IsEngaged:
   type: sensor
   description: Indicates if ESC is currently regulating vehicle stability. True = Engaged. False = Not Engaged.
 
+ESC.IsStrongCrossWindDetected:
+  datatype: boolean
+  type: sensor
+  description: Indicates if ESC system is detecting strong cross winds.
+               True = Strong Cross winds detected.
+               
+ESC.RoadFriction:
+  type: branch
+  description: Road friction values reported by ESC system.
+
+ESC.RoadFriction.MostProbable:
+  datatype: float
```

Comment:
Is this level of precision useful given this is a percentage? That is, would 99.111% be much different from 99%.

Reply:
As always difficult to know what resolution that is needed for future use-cases, float proposed to allow for higher precision than int. Of course, the difference between 99.1% and 99% is not that big, but the relative difference between 0.4% and 0% is bigger. And as with all VSS-signals, the actual accuracy/precision used in transmission can vary between implementations.